framework/sandstone_tests.h

```diff
@@ -27,14 +27,21 @@ extern struct test __stop_tests;
 extern struct test mce_test;
 
 struct test_set_cfg {
-    bool ignore_unknown_tests;  /* whether an error should be reported if
-                                   there's no test matching the specified name,
-                                   either on command-line or in a test list. */
-    bool randomize;             /* iterate through the tests in random order. */
-    bool cycle_through;         /* start over when an iteration through the test
-                                   set has finished. */
-    bool is_selftest;           /* whether to use regular or selftests as the
-                                   source of the tests. */
+    bool ignore_unknown_tests;          /* whether an error should be reported if
+                                           there's no test matching the specified name,
+                                           either on command-line or in a test list. */
+    bool randomize;                     /* iterate through the tests in random order. */
+    bool cycle_through;                 /* start over when an iteration through the test
+                                           set has finished. */
+    bool is_selftest;                   /* whether to use regular or selftests as the
+                                           source of the tests. */
+    bool enable_by_feature_req;         /* whether to enable tests by the requested
```

Comment:
You do not need separate boolean. Instead the default for `enabled_features_mask` should be `1 << 64 - 1` (for 64 bit value). Likewise for disabled.

Reply:
Working on it